srcmedia/ts/members-search.ts

```diff
@@ -47,6 +49,7 @@ document.addEventListener('DOMContentLoaded', () => {
     const birthDateFacet = new RxRangeFilter($birthDateFacet)
     const nationalityFacet = new RxTextFacet($nationalityFacet)
     const arrondissementFacet = new RxTextFacet($arrondissementFacet)
+    const activeFilters = new ActiveFilters($activeFilters)
```

Comment:
Needs to be added to the book search also.

Reply:
thanks for flagging, I definitely might have forgotten otherwise!!